dhcpv6/modifiers.go

```diff
@@ -77,6 +77,21 @@ func WithIANA(addrs ...OptIAAddress) Modifier {
 	}
 }
 
+// WithIAID updates an OptIANA option with the provided IAID
+func WithIAID(iaid [4]byte) Modifier {
```

Comment:
I find this a little bit confusing, both `WithIANA` and `WithIAID` may create a IA_NA option.

Reply:
I know, and I did it because WithIANA is confusing to me in the first place - it doesn't change IANA, but it's a shortcut to extend the IAAddress options within the IANA option. I think both use cases are good, and the documentation should be enough to avoid misunderstandings

Reply:
the alternative in my opinion is to rename WithIANA to WithIAAddrs. WithIAID itself does what it says - it sets the IAID

Reply:
Its fine, lets leave it like that